feat: Add randomness to facts pages

tofumatt opened the pull request:
This adds randomness, as well as previous/next functionality, to every fact page.

It also moves the Facts to a separate folder, in case we want pages in the future (and to more easily query just the "facts" content).

The buttons needed a `navigation` property set, which means they now look a bit weird:

<img width="519" alt="Screenshot 2019-10-08 02 57 26" src="https://user-images.githubusercontent.com/90871/66361667-80098b80-e977-11e9-9e98-9d40d2e20238.png">

That could be a Nautilus thing; if you know of an easy fix feel free to apply it, but if you need me to add an `__unstyled` prop to the `<Button>` component (same as we have for `<Link>`), just let me know and I can do that 😄

tofumatt committed:
feat: Add randomness to facts pages

tofumatt committed:
Merge branch 'master' into feat/random-prev-next

tofumatt committed:
Merge branch 'master' into feat/random-prev-next

tofumatt committed:
fix: Use numbers for sort

sarahmonster commented:
This seems like a good approach! (I am also down with having all the facts in their own folder; makes total sense.)

One thing: the index page now shows an error:
> TypeError: allFactSlugs is undefined

Could we instead have it show a random fact?

sarahmonster committed:
fix: Use separate templates for pages and facts.

This implements a PageWrapper component for the reusable styles, but 
uses slightly different styling depending on the type of content being 
shown. Also fixes the issue where pages were showing an error.

sarahmonster committed:
feat: Create a standalone navigation component.

sarahmonster committed:
feat: Improve overall design of navigation.

sarahmonster committed:
fix: Make sure navigation items line up properly.

tofumatt reviewed (commented):
All looks good to me! ~Wanna get this in, and then I'll add randomness to the homepage button?~

I'll get it merged! 🎉
